tests/integration/_validator/testcases/test_validator.go

```diff
@@ -445,6 +501,7 @@ func (t *ValidatorTest) vspherePluginValues(ctx *test.TestContext, vals []string
 		"n",                                 // add more hosts
 		"n",                                 // add more validation rules
 		"y",                                 // Check role privileges
+		"y",                                 // add another role privilege rule // TODO: fix this, shouldn't be asking for another rule yet
```

Comment:
its bad ux to need to say "yes" again here. my goal for now was to get the tests passing and document whats broken so id rather keep the fix out of scope for this PR.

Reply:
Pretty sure this is only out of order / unexpected because of the aforementioned `DISABLE_KIND_CLUSTER_CHECK` issue.